src/sentry/utils/snuba.py

```diff
@@ -70,6 +70,13 @@
     'stack.colno': 'exception_frames.colno',
     'stack.lineno': 'exception_frames.lineno',
     'stack.stack_level': 'exception_frames.stack_level',
+
+    # misc
+    'release': 'tags[sentry:release]',
+    'message': 'message',
+    'timestamp': 'timestamp',
+    'type': 'type',
+    'platform': 'platform',
```

Comment:
I added these ones that are the same in each because i'm assuming if it's not in this list, it should be converted into `tags[<tag_key>]`